builds/jenkins-build-sls-app/Jenkinsfile

```diff
@@ -169,6 +169,20 @@ node() {
       }
       events.sendCompletedEvent('VALIDATE_PRE_BUILD_CONF', 'pre-build validation completed', context_map, environment_logical_id)
     }
+    stage('Install Plugins') {
+      events.sendStartedEvent('INSTALL_PLUGINS', 'Plugin Installation started', context_map, environment_logical_id)
```

Comment:
Make sure the event is available in events table and event module

Reply:
Yes, it is updated in the event's module.